content/Developer_Program/WebAuthn_Starter_Kit/Security_and_Privacy_Considerations.adoc

```diff
@@ -113,4 +113,4 @@ link:https://w3c.github.io/webauthn/#sctn-username-enumeration[§14.6.2. Usernam
 _If, when initiating an 
 link:https://w3c.github.io/webauthn/#authentication-ceremony[authentication ceremony], there is no account matching the provided username, continue the ceremony by invoking `navigator.credentials.get()` using a syntactically valid `PublicKeyCredent ialRequestOptions` object that is populated with plausible imaginary values._
 
-This recommendation is relevant for the WebAuthnKit for the Server-Verified flow--however, this is not yet supported by the WebAuthnKit for the Server-Verified flow. Currently, the Server-Verified PIN authentication step is supported by the WebAuthnKit, which to some extent mitigates the risk of an attacker trying to list the credentials. To be fully protected against this risk, changes need to be implemented at the WebAuthn server level, which in turn can be integrated into the WebAuthnKit back-end.
+This recommendation is relevant for the WebAuthnKit for the U2F with Verification flow--however, this is not yet supported by the WebAuthnKit for the U2F with Verification flow. Currently, the U2F Password authentication step is supported by the WebAuthnKit, which to some extent mitigates the risk of an attacker trying to list the credentials. To be fully protected against this risk, changes need to be implemented at the WebAuthn server level, which in turn can be integrated into the WebAuthnKit back-end.
```

Comment:
Double check the accuracy of this statement, to what extent does the starter kit mitigate the risk of an attack listing credentials?